tools/tpm2_ptool.py

```diff
@@ -1233,6 +1283,206 @@ def __call__(self, args):
 
                 print('Added key as label: "{keylabel}"'.format(keylabel=keylabel))
 
+@commandlet("importrsa")
+class ImportRsaCommand(Command):
```

Comment:
just a commandlet called *import* or *importkey* with an option for specifying algorithm. Just like *addkey* command. I wonder if it would also be advantageous to have a *---import* option to the addkey command.... probably not.

Reply:
I'll rename it to import. I added rsa because that's only algorithm import currently supports.